aten/src/ATen/cpu/vec256/vec256_complex_double_no_vec.h

```diff
@@ -0,0 +1,328 @@
+#pragma once
+
+#include <ATen/cpu/vec256/intrinsics.h>
+#include <ATen/cpu/vec256/vec256_base.h>
+#if defined(__AVX__) && !defined(_MSC_VER)
+#include <sleef.h>
+#endif
+
+namespace at {
+namespace vec256 {
+// See Note [Acceptable use of anonymous namespace in header]
+namespace {
+
+template <> class Vec256<std::complex<double>> {
```

Comment:
For the record (repeating from the previous PR), we should not have these headers duplicating implementations. I'll make a call on whether or not this should block this PR after reading through the duplication.